pkg/network/setup/podnic.go

```diff
@@ -126,7 +128,17 @@ func newPodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, handler netd
 }
 
 func (l *podNIC) setPodInterfaceCache() error {
-	ifCache := &cache.PodIfaceCacheData{Iface: l.vmiSpecIface}
+	ifCache, err := cache.ReadPodInterfaceCache(l.cacheCreator, string(l.vmi.UID), l.vmiSpecNetwork.Name)
+	if err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			log.Log.Reason(err).Errorf("failed to read pod interface cache for %s", l.vmiSpecNetwork.Name)
```

Comment:
I find the log redundant, the handler controller will eventually print the error like for all the other errors returned by this method.

Reply:
Done